### DISCUSSION TITLE
Gatsby Cloud Status Badge

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [gatsbyjs/gatsby#37100] [Ideas / Feature Requests] [answered]
I would like to have a [status badge](https://docs.netlify.com/monitor-sites/status-badges/) for Gatsby Cloud deployments like the ones that Netlify offers (most often inserted into the `readme.md` files)... I googled around but did not find anything. Has anyone built something like this already? Or could GatsbyJS release this feature?

**Comment by LekoArts** [comment, marked as answer]
Please upvote/comment on https://portal.gatsbyjs.com/gatsby-cloud/p/shields

**Reply by Isaac-Tait** [reply]
Thank you, I will 😄
